app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -123,6 +123,16 @@ class PatientSummaryUpdate(
         event.wasBPMeasuredWithin90Days &&
         areStatinsPrescribedAlready.not()
 
+    val isEligibleForNonLabBasedCvdRisk =
+        event.age in minAgeForStatin..maxAgeForCVDRisk &&
+            isPatientStatinNudgeV2Enabled &&
+            canPrescribeStatin
+
+    val shouldReCalculateCVDRisk =
```

Comment:
Let's name it `shouldCalculateCVDRisk` since we are not just re-calculating it right?